integration/src/main/resources/serviceaccounts/testrunner-sa.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "testrunner-perf@terra-kernel-k8s.iam.gserviceaccount.com",
+  "name": "testrunner-perf@broad-dsde-perf.iam.gserviceaccount.com",
```

Comment:
Maybe this file should be renamed to `testrunner-perf` instead of `testrunner-sa`?

Reply:
It has been renamed to `testrunner-perf`.